howso/utilities/utilities.py

```diff
@@ -1148,11 +1173,11 @@ def deep_update(base, updates):
 def matrix_processing( # noqa
     matrix: pd.DataFrame,
     normalize: bool = False,
-    normalize_method: Union[Iterable[Union[str, Callable]], str, Callable] = "relative",
+    normalize_method: Iterable[str | Callable, str, Callable] = "relative",
```

Comment:
This should probably be:

`normalize_method: Iterable[t.Literal["relative", "fractional", "feature_count"] | Callable] | t.Literal["relative", "fractional", "feature_count"] | Callable = "relative",`